var/spack/repos/builtin/packages/rivet/package.py

```diff
@@ -106,16 +106,16 @@ class Rivet(AutotoolsPackage):
     depends_on('yoda@1.7.0:1.7', when='@2.6.0,2.7.0,2.7.1,3.0.0,3.0.2')
     depends_on('yoda@1.5.0:1.5', when='@2.4.1')
 
-    depends_on('hepmc',  type=('build', 'link', 'run'), when='hepmc=2')
-    depends_on('hepmc3', type=('build', 'link', 'run'), when='hepmc=3')
-    depends_on('boost', when='@:2.5.0', type=('build', 'run'))
-    depends_on('fastjet', type=('build', 'run'))
-    depends_on('fjcontrib', type=('build', 'run'), when='@3.0.0:')
-    depends_on('gsl', type=('build', 'run'), when='@:2.6.0,2.6.2:2')
+    depends_on('hepmc', when='hepmc=2')
+    depends_on('hepmc3', when='hepmc=3')
+    depends_on('boost', when='@:2.5.0')
+    depends_on('fastjet')
+    depends_on('fjcontrib', when='@3.0.0:')
+    depends_on('gsl', when='@:2.6.0,2.6.2:2')
```

Comment:
I don't think GSL should be a runtime dependency either: the only ones that need it are imported python bindings right?

Reply:
I'm not sure about that either - I guess also if a package makes use of any executables of dependencies setup via path. I don't think this is the case for gsl, but it certainly should be `('build','link')`

Reply:
OK, it shouldn't hurt to leave it aside from polluting the runtime environment.